apps/tests/test_forms.py

```diff
@@ -44,6 +44,7 @@ def setUp(self):
             "port": 8000,
             "image": "ghcr.io/scilifelabdatacentre/image:tag",
             "tags": ["tag1", "tag2", "tag3"],
+            "custom_default_url": "valid-custom_default_url/",
```

Comment:
I'd call it a `default_url_subpath`, but I'm not insisting. Just that it's not a full URL, that's bothering me a little.

Reply:
Rename this field to `default_url_subpath` all together.